code/modules/power/smes.dm

```diff
@@ -208,8 +208,6 @@
 		disconnect_terminal()
 	return ..()
 
-	return round(5.5*charge/(capacity ? capacity : 5e6))

```

Comment:
I'm...not sure why/when I did this, but I also took a multi-month break when working on this. I think I just found a useless line and deleted this. Can move this to an unrelated PR if desired.